.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "cpp/third_party/objecthash"]
```

Comment:
We already have a `cpp/third_party` dir for the hostname checking stuff so I just put it in the same place, but it's a good point about the other languages, though, so I've moved it up and created a top-level `third_party`.